ButtonFunctions/__init__.py

```diff
@@ -19,3 +23,35 @@ def save_curve(application) -> None:
         wr = csv.writer(myfile, quoting=csv.QUOTE_ALL)
         wr.writerow(application.times)
         wr.writerow(application.forces)
+
+
+def reset(application):
+    prompt = application.add_prompt("Warning!", "This will erase all unsaved data. Are you sure?")
+    if prompt:
+        # application.connection.reset()
+        application.data_initialization()
+        for key, item in application.plot_pannels.items():
+            item.clear()
+
+
+
+
+
+def start_transducer(application):
+    sensor = application.get_sensor_info(PRESSURE_TRANSDUCER_REF)
```

Comment:
Can we make this kind of switch generic? Because we are going to have a lot of functions similar to those

Reply:
Great idea. I thought about it a while ago.
Will work on it.